lib/command.c

```diff
@@ -592,6 +589,9 @@ static int config_write_host(struct vty *vty)
 
 		if (host.motdfile)
 			vty_out(vty, "banner motd file %s\n", host.motdfile);
+		else if (strncmp(host.motd, FRR_DEFAULT_MOTD,
```

Comment:
need to check if `host.motd` is null or not before dereferencing it

Reply:
fixed